src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1068,15 +1069,19 @@ private bool HasHeader
         /// <param name="hasHeader">Whether the file has a header.</param>
         /// <param name="separatorChar"> The character used as separator between data points in a row. By default the tab character is used as separator.</param>
         /// <param name="dataSample">Allows to expose items that can be used for reading.</param>
-        internal TextLoader(IHostEnvironment env, Column[] columns, bool hasHeader = false, char separatorChar = '\t', IMultiStreamSource dataSample = null)
-            : this(env, MakeArgs(columns, hasHeader, new[] { separatorChar }), dataSample)
+        /// <param name="allowSparse">Whether the file can contain numerical vectors in sparse format.</param>
+        /// <param name="allowQuoting">Whether the file can contain numerical vectors in sparse format.</param>
```

Comment:
copy-paste error here. I don't think this is what `allowQuoting` means :)

Reply:
Thanks. Got it changed to `/// <param name="allowQuoting">Whether the content of a column can be parsed from a string starting and ending with quote.</param>`.


---
In reply to: [259418380](https://github.com/dotnet/machinelearning/pull/2630#discussion_r259418380) [](ancestors = 259418380)